tools/flaky-test-reporter/result.go

```diff
@@ -49,9 +49,10 @@ type RepoData struct {
 
 // JobConfig is initial configuration for a given repo, defines which job to scan
 type JobConfig struct {
-	Name string
-	Repo string
-	Type string
+	Name      string
```

Comment:
we may need to rethink about the naming of `RepoData` and `JobConfig` since the first one doesn't make that much sense any more. We can discuss about this offline

Reply:
The most logical change will require more refactoring, will address in a future PR